### ISSUE TITLE
Install and configure audit log plugin

### ISSUE THREAD

**Issue by artlogic** [wdetdigital/wdet-conversion#163]

**Comment by Fw7424** [comment]
https://wordpress.org/plugins/simple-history/ 
Some of the highlights are side-by-side change comparison and user/type filtering

**Comment by artlogic** [comment]
Installed in dev

**Comment by artlogic** [comment]
@Fw7424 please let me know what you find about creating a backend log using this plugin - thanks.

**Comment by Fw7424** [comment]
> 

https://anchor.host/email-triggers-for-simple-history/

**Comment by Fw7424** [comment]
Ryan signed off, with pending questions about if we could use multiple emails. I think we could use a foreach loop for the things that need triggers. Looks like a trigger will need to be set in this code.

**Comment by artlogic** [comment]
> Ryan signed off, with pending questions about if we could use multiple emails. I think we could use a foreach loop for the things that need triggers. Looks like a trigger will need to be set in this code.

Agreed, however it looks like we'll need to identify which message types to send emails on - I'm pretty sure it'll get too noisy if we send on every type.
